bot/interact.go

```diff
@@ -149,7 +149,7 @@ func validateSubmission(messageJobLink, messageMaxSalary, messageMinSalary strin
 		}
 
 		if 2.5*float32(minSalary) < float32(maxSalary) {
-			validationErrors["max_salary"] = "The gap between MinSalary and MaxSalary is rather large. Maybe you should post two different job offers with different responsibilities and required qualifications. Salary is a relevant field, we recommend you try to keep it meaningful to increase the chances of taking the position seriously by potential candidates."
+			validationErrors["max_salary"] = "The range MinSalary-MaxSalary is too large. Salary is a relevant field, keep it meaningful to increase offer impact."
```

Comment:
Some suggestions for a more readable version IMHO:
```suggestion
			validationErrors["max_salary"] = "The range min-max salary is too wide. Salary is a relevant field; keep it meaningful to increase offer impact."
```